Geometry/HGCalGeometry/plugins/HGCalGeometryESProducer.cc

```diff
@@ -30,7 +30,7 @@
 #include "Geometry/HGCalGeometry/interface/HGCalGeometry.h"
 #include "Geometry/HGCalGeometry/interface/HGCalGeometryLoader.h"
 
-//#define DebugLog
+#define DebugLog
```

Comment:
@bsunanda - it's better to use LogDebug which can be switched on/off without editing the code. In any case, this line should be commented out when it's submitted for a release.